modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -183,6 +263,7 @@ const neoSchema = new Neo4jGraphQL({ features, typeDefs, driver });
 
 For both `String` and `ID`:
 
+// Is the following correct? would have expected "filters: { String : { MATCHES: true, ID: true}}" based on the two above
```

Comment:
^

Reply:
No, the above is actually incorrect.

Correct examples:

```ts
features: {
    filters: {
        ID: {
            MATCHES: true,
        },
    },
},
```


```ts
features: {
  filters: {
      String: {
          MATCHES: true,  
      },
      ID: {
          MATCHES: true,
      },
  },
},
```